### PR TITLE
fix: Capitalise the first letter of the first word in the emoji name displayed on the right-side panel in the message details.

### PR DESCRIPTION
**Pull request by arjita-mitra**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - Capitalise the first letter of the first word in the emoji name displayed on the right-side panel in the message details.

- The **PR Description**
  - Capitalise the first letter of the first word in the emoji name displayed on the right-side panel in the message details.
----
##### References
1. https://wearezeta.atlassian.net/browse/WPB-3293

### CODE REVIEW

File: src/script/page/RightSidebar/MessageDetails/MessageDetails.tsx [hunk 1]
```diff
@@ -229,6 +229,13 @@ const MessageDetails: FC<MessageDetailsProps> = ({
 
   const onParticipantClick = (userEntity: User) => togglePanel(PanelState.GROUP_PARTICIPANT_USER, userEntity);
 
+  function capitalizeFirstLetter(emojiName: string) {
+    if (emojiName && emojiName.trim() !== '') {
+      return emojiName.charAt(0).toUpperCase() + emojiName.slice(1);
```
Comment (PatrykBuniX) [inline src/script/page/RightSidebar/MessageDetails/MessageDetails.tsx]: We have `capitalizeFirstChar` method in our `StringUtil.ts` let's use it

Comment (arjita-mitra) [reply src/script/page/RightSidebar/MessageDetails/MessageDetails.tsx]: I want to perform the sanity check.

if (emojiName && emojiName.trim() !== '') 

shall i add it to the capitalizeFirstChar method in StringUtil.ts?

Comment (PatrykBuniX) [reply src/script/page/RightSidebar/MessageDetails/MessageDetails.tsx]: No, we can do just `return capitalizeFirstChar(emojiName)` after sanity check is valid - we will need to rename the original method then.

Comment (arjita-mitra) [reply src/script/page/RightSidebar/MessageDetails/MessageDetails.tsx]: why not adding it to the capitalizeFirstChar method? this is a basic sanity check, it should be there in the method.

**Review by PatrykBuniX — APPROVED** [review]
👌

**Pull request merged**